exchange_calendars/calendar_helpers.py

```diff
@@ -71,3 +82,96 @@ def compute_all_minutes(
             )
     out = np.concatenate(pieces).view("datetime64[ns]")
     return out
+
+
+def parse_session(
```

Comment:
Is the a reason not to put this as a function on the `ExchangeCalendar` class? Seems like you're only using it there and always passing `self`.

Reply:
I envisaged using it to parse `start` and `end` parameters received to ExchangeCalendarDispatcher.get_calendar (#31). To this end I've made revisions to allow `calendar` to take None if `strict` is False.